library/x509_crt.c

```diff
@@ -1435,32 +1571,179 @@ int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path )
 }
 #endif /* MBEDTLS_FS_IO */
 
+/*
+ * OtherName ::= SEQUENCE {
+ *      type-id    OBJECT IDENTIFIER,
+ *      value      [0] EXPLICIT ANY DEFINED BY type-id }
+ *
+ * HardwareModuleName ::= SEQUENCE {
+ *                           hwType OBJECT IDENTIFIER,
+ *                           hwSerialNum OCTET STRING }
+ *
+ * NOTE: we currently only parse and use otherName of type HwModuleName,
+ * as defined in RFC 4108.
+ */
+int mbedtls_x509_get_other_name( const mbedtls_x509_sequence *subject_alt_name,
+                                 mbedtls_x509_san_othername *other_name )
```

Comment:
Perhaps the type should be mbedtls_x509_san_other_name instead of `mbedtls_x509_san_othername`?

Reply:
I don't mind, but the RFC calls it `otherName`, so I wanted it to be closer to the rfc naming

Reply:
I believe the RFC uses capitalization, that is why there is an N rather than n. We use _ (underscore), so thats why I suggest using other_name instead of othername. For example, we have sequence_of somewhere in the Mbed TLS code as opposed to sequenceof